dsn.go

```diff
@@ -702,6 +707,13 @@ func parseDSNParams(cfg *Config, params string) (err error) {
 			cfg.Tracing = value
 		case "tmpDirPath":
 			cfg.TmpDirPath = value
+		case "disableQueryContextCache":
+			var b bool
+			b, err = strconv.ParseBool(value)
+			if err != nil {
+				return
```

Comment:
don't we want to inform user that he passed wrong type of value?

Reply:
Actually we do. This is called "naked" return and it means, that we return the variable with the same name as in function declaration.